### PR TITLE
add istioctl info to help user easily adopt Istio

### PR DESCRIPTION
**Pull request by irisdingbj**

### CODE REVIEW

File: content/docs/setup/kubernetes/install/kubernetes/index.md [hunk 1]
```diff
@@ -28,6 +28,15 @@ This permits customization of Istio to operator specific requirements.
 
 1. Perform any necessary [platform-specific setup](/docs/setup/kubernetes/prepare/platform-setup/).
 
+    {{< tip >}}
+    Run following command to verify the Kubernetes cluster is ready for Istio installation:
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/install/kubernetes/index.md]: Run the following

File: content/docs/setup/kubernetes/install/kubernetes/index.md [hunk 2]
```diff
@@ -28,6 +28,15 @@ This permits customization of Istio to operator specific requirements.
...
+
+    {{< text bash >}}
+    $ istioctl verify-install
```
Comment (frankbu) [inline content/docs/setup/kubernetes/install/kubernetes/index.md]: Can you add something to indicate what is the expected output?

File: content/docs/setup/kubernetes/install/kubernetes/index.md [hunk 3]
```diff
@@ -137,6 +146,12 @@ $ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
     prometheus-67cdb66cbb-9w2hm                                    1/1     Running     0          1m
     {{< /text >}}
 
+1.  Run the following command to verify the installation succeeds:
+
+    {{< text bash >}}
+    $ istioctl verify-install -f install/kubernetes/istio-demo.yaml
```
Comment (frankbu) [inline content/docs/setup/kubernetes/install/kubernetes/index.md]: Does user need to do the previous 2 steps and also this? Isn't this sufficient to verify?

File: content/docs/setup/kubernetes/install/kubernetes/index.md [hunk 4]
```diff
@@ -147,6 +162,15 @@ The application must use either the HTTP/1.1 or HTTP/2.0 protocols for all its H
 traffic; HTTP/1.0 is not supported.
 {{< /warning >}}
 
+{{< tip >}}
+Run the following command to validate your application:
+
+{{< text bash >}}
+$ istioctl validate -f <your-app-spec>.yaml
+{{< /text >}}
```
Comment (frankbu) [inline content/docs/setup/kubernetes/install/kubernetes/index.md]: Expected output?

**Comment by frankbu** [comment]
Shouldn't these instructions also be added to the helm installation instructions?

**Comment by tangiel** [comment]
@googlebot rescan

File: content/docs/setup/kubernetes/install/kubernetes/index.md (hunk 1 above)
Comment (irisdingbj) [reply content/docs/setup/kubernetes/install/kubernetes/index.md]: done

File: content/docs/setup/kubernetes/install/kubernetes/index.md (hunk 2 above)
Comment (irisdingbj) [reply content/docs/setup/kubernetes/install/kubernetes/index.md]: done

File: content/docs/setup/kubernetes/install/kubernetes/index.md (hunk 4 above)
Comment (irisdingbj) [reply content/docs/setup/kubernetes/install/kubernetes/index.md]: done

File: content/docs/setup/kubernetes/install/kubernetes/index.md (hunk 3 above)
Comment (irisdingbj) [reply content/docs/setup/kubernetes/install/kubernetes/index.md]: there are some tips about how to access the gateway in the previous 2 steps. And these two additional check do not harm. So can we keep it here?

Comment (irisdingbj) [reply content/docs/setup/kubernetes/install/kubernetes/index.md]: I put the short output here as the verbose version will have really long output.

**Comment by geeknoid** [comment]
A few CI failures:

./content/docs/setup/kubernetes/install/kubernetes/index.md:61: MD009 Trailing spaces

And the bad link errors can be addressed by rebasing.

Thanks.

**Comment by geeknoid** [comment]
@irisdingbj Do you have time to push this over the finish line?

Thanks.

**Comment by geeknoid** [comment]
@irisdingbj Ping.

**Comment by geeknoid** [comment]
Closing. Please reopen if you have time to finish this.

**Pull request closed without merging**